tutorial/pangeo101/xarray_introduction.ipynb

```diff
@@ -18,11 +18,11 @@
     "### Authors\n",
```

Comment:
<p>Do we talk about nvdi here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-data/clivar-2022/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='85'/>